<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDosya">
    <property name="title">
     <string>Dosya</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionYeni"/>
    <addaction name="actionDisaAktar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuEkranlar">
    <property name="title">
     <string>Ekranlar</string>
    </property>
    <addaction name="actionAnaliz_Ekran"/>
   </widget>
   <addaction name="menuDosya"/>
   <addaction name="menuEkranlar"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionYeni">
   <property name="text">
    <string>Yeni Analiz</string>
   </property>
  </action>
  <action name="actionDisaAktar">
   <property name="text">
    <string>Dışa Aktar</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Yardım</string>
   </property>
  </action>
  <action name="actionAnaliz_Ekran">
   <property name="text">
    <string>Analiz Ekranı</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
